[CS2113-T13-4] TypingGame

Isabella-L opened the pull request:
Typing Game is a CLI-based game in which users can type phrases under different modes. After each game round, users will be able to view a summary report of their performance.

LimJiaJing committed:
Merge pull request #56 from Isabella-L/v2.0-timermode

Some updates after meeting

Isabella-L committed:
modifications and printout userInput

prints out the userInput for timeModeGame as per request by JiaJing

Isabella-L committed:
checkstyle update

Isabella-L committed:
Update EXPECTED.TXT

LimJiaJing committed:
Merge pull request #60 from Isabella-L/v2.0-timermode

modifications and printout userInput

LimJiaJing committed:
Initial change for revamp for summary feature

LimJiaJing committed:
Merge branch 'master' into redo-summary-feature

# Conflicts:
#	src/main/java/seedu/typists/command/ContentCommand.java

LimJiaJing committed:
Change signature of some methods in SummaryManager and SummaryUI.

Isabella-L committed:
more OOP for game class

LimJiaJing committed:
Debug summary feature, Link summary feature to game, create some classes for future use.

Create GameRecordManager and GameRecord for storage feature.

LimJiaJing committed:
Redo logic of calculating error words, error word counts, and correct word counts in SummaryManager

Isabella-L committed:
Merge pull request #61 from LimJiaJing/redo-summary-feature

Redo summary feature

LimJiaJing committed:
Add test code for SummaryUi and SummaryManager

LimJiaJing committed:
Reformat code to abide by coding standard

DuckWillow committed:
Merge pull request #62 from LimJiaJing/redo-summary-feature

Add test code for SummaryUi and SummaryManager

LimJiaJing committed:
First commit of storage feature (untested)

LimJiaJing committed:
Edit Command interface to accept command arguments. Add ViewCommand.

Make required changes to pass CLI checks

Isabella-L committed:
Merge pull request #63 from LimJiaJing/edit-command-interface

Edit Command interface to accept command arguments. Add ViewCommand.

LimJiaJing committed:
Merge branch 'master' into storage-v2.0

LimJiaJing committed:
Convert methods in Storage to static. Associate Storage to GameRecordsManager.

DuckWillow committed:
Some DG update

LimJiaJing committed:
First commit of view feature (untested)

LimJiaJing committed:
Ignore seedu.typists.ui.TextUi.log

LimJiaJing committed:
commit log file

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into v2.0-timermode

LimJiaJing committed:
Add automatic store and read game records feature

Fixes #41
Fixes #40

Isabella-L committed:
fix bug

Isabella-L committed:
minor

LimJiaJing committed:
Edit exit logic

LimJiaJing committed:
Merge pull request #67 from Isabella-L/v2.0-timermode

Fixing compare data for game

LimJiaJing committed:
Merge pull request #65 from DuckWillow/master

Some DG update

LimJiaJing committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp

# Conflicts:
#	seedu.typists.ui.TextUi.log

LimJiaJing committed:
Update proposedImplementation for View Statistics Feature

snuckerzlol committed:
Remove animation and error, DG update

snuckerzlol committed:
DG

snuckerzlol committed:
Merge branch 'master' into storage-v2.0

snuckerzlol committed:
Merge pull request #66 from LimJiaJing/storage-v2.0

Storage v2.0

snuckerzlol committed:
Merge branch 'master' into master

snuckerzlol committed:
DG

snuckerzlol committed:
Merge remote-tracking branch 'origin/master'

snuckerzlol committed:
DG

Isabella-L committed:
Update seedu.typists.ui.TextUi.log

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into v2.0-timermode

snuckerzlol committed:
Delete Content.puml

LimJiaJing committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp

# Conflicts:
#	seedu.typists.ui.TextUi.log

LimJiaJing committed:
Merge branch 'master' into view-past-records-v2.0

# Conflicts:
#	src/main/java/seedu/typists/game/GameRecord.java
#	src/main/java/seedu/typists/game/GameRecordsManager.java

SuibianP commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,41 @@
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+* https://www.baeldung.com/java-testing-system-out-println
+
+##Architecture
```
Missing whitespace after `##`, as per Markdown specifications.

nvbinh15 replied:
Missing whitespace after `##`, which makes the line not be rendered correctly on the Github page

<img width="466" alt="Screen Shot 2021-10-28 at 09 07 28" src="https://user-images.githubusercontent.com/62022886/139169020-43091715-f2ae-493d-bd92-ef90cadbfa08.png">

SuibianP commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,41 @@
+
+Gonna to change this .puml diagram later - zhansen
+
+![](uml/WordLimitMode.png)
```
Would it be better to use vector graphics such as SVG, which displays nicely even on the dark background you choose?

It would be better to just link to an online one that renders the diagram. This has the following benefits of reducing binary blobs in the repo.

nvbinh15 replied:
There should be `:` before the class name in the notations of entities. If method calls to `static` methods are received by the class itself, there should be `<<class>>` in the notation.

<img width="627" alt="Screen Shot 2021-10-28 at 09 11 10" src="https://user-images.githubusercontent.com/62022886/139169187-43db29c1-5b32-4c9d-94a7-c2f0186a1a45.png">

LimJiaJing committed:
Finish view history feature.

Rename ViewCommand and ViewCommandUi class to HistoryCommand and HistoryCommandUi respectively
Add a constructor with a String argument to InvalidCommandException
Add history as a command to the game
Debug summary feature
Fixes #6
Fixes #39

LimJiaJing committed:
Remove IO redirection test from CLI

LimJiaJing committed:
Add a println at exit command to inform user the program is updating the records

LimJiaJing committed:
Merge branch 'clear-history-records-v2.0' into view-past-records-v2.0

DuckWillow committed:
Merge pull request #71 from LimJiaJing/view-past-records-v2.0

View past records v2.0

LimJiaJing committed:
Add ClearCommand class

LimJiaJing committed:
Merge branch 'master' into clear-history-records-v2.0

LimJiaJing committed:
Complete clear records feature

Fixes #38

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into v2.0-timermode

Isabella-L committed:
Merge pull request #72 from LimJiaJing/clear-history-records-v2.0

Clear history records v2.0

LimJiaJing committed:
Merge branch 'master' into update-DG

LimJiaJing committed:
Merge branch 'master' into update-DG

Isabella-L committed:
TimerMode Enhance

Create gameUi class
delete dataprocessor class
fix bugs

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into v2.0-timermode

Isabella-L committed:
update to pass test

LimJiaJing committed:
Edit description of View Statistics feature

LimJiaJing committed:
View Statistics Object Diagram.drawio.png

LimJiaJing committed:
Change style of object diagram

LimJiaJing committed:
Add 3 object diagrams

LimJiaJing committed:
Add three object diagrams. Change to using svg file.

Stop using drawio/mode=github.

LimJiaJing committed:
Add 3 object diagrams into DG

LimJiaJing committed:
Complete object diagram part of View Statistics feature explanation.

Isabella-L committed:
Update parser for timemodegame

Isabella-L committed:
update to pass test

Isabella-L committed:
add start now feature

Isabella-L committed:
checkstyle update

Isabella-L committed:
Update TimeGameCommand.java

LimJiaJing committed:
Merge pull request #73 from Isabella-L/v2.0-timermode

update V2.0 timermode

LimJiaJing committed:
Complete developer guide for View Statistics feature.

#Fixes 68

Isabella-L committed:
fix content supply bug for timemodegame

LimJiaJing committed:
Merge pull request #76 from Isabella-L/start-now-feature

add start now option for time mode game

Isabella-L committed:
minor

Isabella-L committed:
minor

Isabella-L committed:
set content with game feature

Isabella-L committed:
Merge branch 'start-now-feature' into content-supply-bug

Isabella-L committed:
update test result

add log file into gitignore

LimJiaJing committed:
Update UG

Updates:notes about command format, view past records feature, clear feature, saving the data feature, and warning about editing the data file.

LimJiaJing committed:
Change format of warning

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into DG-update

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into content-supply-bug

Isabella-L committed:
Merge pull request #80 from LimJiaJing/update-UG-v2.0

Update ug v2.0

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into DG-update

Isabella-L committed:
Merge pull request #79 from Isabella-L/content-supply-bug

Fix content supply bug for timemode game and add set content feature

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into DG-update

Isabella-L committed:
update game parser

Merge time game parser and word game parser into one

Isabella-L committed:
update UG

nvbinh15 commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,41 @@
 # Developer Guide
```
The sections should be numbered and the table of content should be included.

nvbinh15 commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,41 @@
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+* https://www.baeldung.com/java-testing-system-out-println
```
The link is now plain text. It should be wrapped with `[name](link)` so that it can be clicked.

nvbinh15 reviewed (commented):
Overall, not so much content is included in the DG. A UML sequence diagram is included to explain the interactions between entities when a `word` command is read; however, there are some minor bugs.

Isabella-L committed:
Update DeveloperGuide.md

Isabella-L committed:
Merge pull request #81 from Isabella-L/DG-update

UG update by Sicheng

Isabella-L committed:
Merge pull request #77 from LimJiaJing/update-DG

Update dg

Isabella-L committed:
add sequence diagram (temporary)

exetr commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,96 @@
+### Time Limit Game
+Sequence Diagram for Time Mode Game:
+![image](https://user-images.githubusercontent.com/69776265/139190231-eb648329-517b-42dc-a088-fbce5c93c616.png)
+
```
It is possible to position the elements such that the text is not blocked by others so it is easier to read?
![image](https://user-images.githubusercontent.com/23265811/139215634-debf514f-1051-4919-9f0b-12c3662222d2.png)

exetr replied:
Similarly, are the return arrows supposed to show return values or method calls? If it is return values, the arrow should be dotted [here](https://nus-cs2113-ay2122s1.github.io/website/schedule/week10/topics.html#w10-1-sequence-diagrams-basics)

exetr commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,96 @@
+
+Step 7. ViewCommand calls the displayStatistics() method of the ViewCommandUi class to display the statistics.
+![Alt test](uml/ViewStatistics-4.drawio.svg)
+
```
![image](https://user-images.githubusercontent.com/23265811/139221164-f77fea05-83c6-455f-99e4-ee3049c246f8.png)
Are the multiple stacked boxes meant to represent multiple objects? If so, can perhaps considering removing them and rely on the multiplicity values instead as it may be quite confusing and redundant.

exetr commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,96 @@
+
+
+The following sequence diagram shows how the above scenario is executed.
+![Alt test](uml/ViewStatistics-5.drawio.svg)
```
![image](https://user-images.githubusercontent.com/23265811/139221697-50dae934-fe07-4292-9a87-57a0a54db6d6.png)
Should there be dotted arrows back to the calling class after each `readGameRecords()` call? Since the activation bar implies that control is handed back to the `GameRecordsManager()` momentarily.

YEOWEIHNGWHYELAB replied:
When calling a constructor, there is no need to put the "new". Moreover you should use the same sequence diagram drawing tool as stated in the module page (attached below).

![Untitled](https://user-images.githubusercontent.com/69447385/139372398-476f91a4-9bb1-4ff3-a6e1-4fa4bfe57d2f.png)
![Untitled](https://user-images.githubusercontent.com/69447385/139372519-4d7449df-f518-4c7e-befb-3ef9d9bd210a.png)

DuckWillow committed:
Merge summary into word limit game

DuckWillow committed:
Merge branch 'master' into branch76

DuckWillow committed:
Merge from master to branch 76

DuckWillow committed:
Debug

DuckWillow committed:
Update Utils.java

Isabella-L committed:
add time game sequence diagram

YEOWEIHNGWHYELAB commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,96 @@
+Sequence Diagram for Time Mode Game:
+![image](https://user-images.githubusercontent.com/69776265/139190231-eb648329-517b-42dc-a088-fbce5c93c616.png)
+
+### \[Proposed\] View Statistics feature
```
The self-invocation of the Sequence diagram is missing the method it is calling.

![Untitled](https://user-images.githubusercontent.com/69447385/139371631-75024442-4bd8-4820-a3ac-915fe0e995cb.png)

YEOWEIHNGWHYELAB commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,96 @@
+
+### Choose the game content
+
+### Word Limit Game
```
Should the "Alt" be changed to "Opt" instead since you do not have the "else" block?

![Untitled](https://user-images.githubusercontent.com/69447385/139372027-fbff4971-1772-470c-8165-88e80f6f36ec.png)

YEOWEIHNGWHYELAB commented on docs/DeveloperGuide.md:
```diff
@@ -1,15 +1,96 @@
+![image](https://user-images.githubusercontent.com/69776265/139190231-eb648329-517b-42dc-a088-fbce5c93c616.png)
+
+### \[Proposed\] View Statistics feature
+#### \[Proposed Implementation\]
```
There seems to be missing dotted arrow heads back to the calling class as I circle in red in the picture attached below. Moreover, your loop and alt format is not drawn correctly, please refer to the module webpage for the details.

![Untitled](https://user-images.githubusercontent.com/69447385/139373100-be1ac7be-951e-4bef-a640-beef827b2ba3.png)

YEOWEIHNGWHYELAB reviewed (commented):
Overall, I understand that you are under time constraint, otherwise I believe that you would be able to do a much better DG especially on the sequence diagram as I could find quite a number of bugs. But you should take the remaining time and review them. 

Good effort tho 👍

Isabella-L committed:
udpate DG

Isabella-L committed:
Merge pull request #86 from DuckWillow/branch76

Merge summary into word limit game

Isabella-L committed:
fix display line problem for time game

Isabella-L committed:
change error msg for timemode command

Isabella-L committed:
add gitignore

Isabella-L committed:
update checkstyle

DuckWillow committed:
Merge pull request #87 from Isabella-L/v2.0-timermode

modify util method

Isabella-L committed:
update sample txt

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into v2.0-timermode

DuckWillow committed:
Improve on word limit game

-remove repeated summary
-remove display text at beginning
-updated UG

Isabella-L committed:
Merge pull request #88 from DuckWillow/branch76

Improve on word limit game

Isabella-L committed:
update

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into v2.0-timermode

LimJiaJing committed:
Address code review comment

LimJiaJing committed:
Add exception handling for non-positive integers user input in history command's number of game records argument.

Fixes #122

LimJiaJing committed:
Add help display for history and clear command

Fixes #102
Fixes #132

LimJiaJing committed:
Remove log of INFO level from terminal

Fixes #118

LimJiaJing committed:
Edit start screen to make usage of clear and history commands clearer.

Editted message such that gamers will know the command syntax for history and clear command
Delete old showSummary() method

LimJiaJing committed:
Edit format of user error messages for clear and history command.

snuckerzlol committed:
Update CommandFactory.java

snuckerzlol committed:
Update TextUi.java

snuckerzlol committed:
Update TextUi.java

snuckerzlol committed:
advad

snuckerzlol committed:
Merge remote-tracking branch 'origin/master'

snuckerzlol committed:
Merge branch 'master' into master

snuckerzlol committed:
Merge pull request #70 from maxwireddevs/master

Remove animation and error, DG update

snuckerzlol committed:
delete error

snuckerzlol committed:
Merge pull request #140 from maxwireddevs/master

delete error

snuckerzlol committed:
Merge pull request #139 from LimJiaJing/address-bugs-ped

Address bugs ped

LimJiaJing committed:
Change application to not crash when record file is editted to an invalid format.

Update user guide to fit the change in logic.

Fixes #115

snuckerzlol committed:
Merge pull request #141 from LimJiaJing/address-bugs-ped

Change application to not crash when record file is edited

LimJiaJing committed:
Fix bug

str1==str2 to str1.equals(str2)

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into v2.0-timermode

Isabella-L committed:
move getReady method to superclass game

Isabella-L committed:
more OOP

add seperate commands for the 2 game,
if they specify number inline with the command, no get Limit promt.

LimJiaJing committed:
Improve 'clear' command logic

Clears file immediately after gamer inputs command (previously clears file only after gamer enters the 'bye' command

LimJiaJing committed:
Improve format of error message to user in history command

LimJiaJing committed:
Improve user guide

Reformat user guide
add summary feature
explain command format more clearly

Isabella-L committed:
improve command parsing

add seperate method in util to analyse if the time limit given is valid.
now user can specify the time/word limit of game in either the command or into the game

LimJiaJing committed:
Format help message for history and clear command

Isabella-L committed:
pass test

LimJiaJing committed:
Correct developer guide uml images for view statistics feature

Correct representation of array on object diagram
Correct representation of static method in sequence diagram
Fixes #85

DuckWillow committed:
Merge pull request #142 from LimJiaJing/address-bugs-ped

Address bugs ped

DuckWillow committed:
Merge pull request #143 from LimJiaJing/user-guide-v2.1

Improve user guide

DuckWillow committed:
Merge pull request #144 from Isabella-L/v2.0-timermode

V2.1 timermode fix bug

DuckWillow committed:
Merge pull request #145 from LimJiaJing/developer-guide-v2.1

Correct developer guide uml images for view statistics feature

DuckWillow committed:
Fix word mode negative number

DuckWillow committed:
Update -w to -word

LimJiaJing committed:
Add content in PPP.

DuckWillow committed:
Merge pull request #147 from LimJiaJing/ppp

limjiajing-Add content in PPP.

Isabella-L committed:
Merge pull request #146 from DuckWillow/branch76

Fix word mode negative number

LimJiaJing committed:
Fix github pages rendering issues trial 1

LimJiaJing committed:
Fix github pages rendering issue trial 1

LimJiaJing committed:
Fix github pages rendering issue trial 2

LimJiaJing committed:
fix github pages rendering issue trial 2

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into DG-update

Isabella-L committed:
update typist

LimJiaJing committed:
Add user stories

LimJiaJing committed:
Edit ppp format for pdf conversion

LimJiaJing committed:
Edit ppp to fix github rendering trial 1

LimJiaJing committed:
edit ppp format to fix github pages rendering issue

snuckerzlol committed:
Merge pull request #148 from LimJiaJing/ppp

Fix github pages rendering issue trial 1

snuckerzlol committed:
Merge pull request #150 from LimJiaJing/developer-guide-v2.1

Add user stories

LimJiaJing committed:
Edit format to fix github pages rendering issue

LimJiaJing committed:
Change RepoSense link to automatically how lim jia jing contributions

Isabella-L committed:
update UG and fix -c bug

Isabella-L committed:
Update UserGuide.md

Isabella-L committed:
Update TextUi.java

Isabella-L committed:
Merge pull request #151 from LimJiaJing/ppp

Edit ppp format to fit github pages and pdf conversion

Isabella-L committed:
Merge branch 'master' into user-guide-v2.1

Isabella-L committed:
Merge pull request #149 from LimJiaJing/user-guide-v2.1

Fix github pages rendering issues trial 1

Isabella-L committed:
Merge branch 'master' into DG-update

LimJiaJing committed:
Merge pull request #152 from Isabella-L/DG-update

UG update and fix bugs

Isabella-L committed:
Update UserGuide.md

Isabella-L committed:
Merge branch 'DG-update' of https://github.com/Isabella-L/tp into DG-update

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into DG-update

Isabella-L committed:
update UG

Isabella-L committed:
rename uml folder into diagram

Isabella-L committed:
delete MANIFEST, update DG

Isabella-L committed:
delete data folder and gamerInfo

Isabella-L committed:
move exception folder into common foler

Isabella-L committed:
refactor packages

move StringParser class to common folder.
delete the parser folder which only had StringParser class inside.

snuckerzlol committed:
Updates

Isabella-L committed:
rename TimeLimitGmae

Isabella-L committed:
Merge pull request #154 from maxwireddevs/master

Updates

Isabella-L committed:
Merge branch 'master' into DG-update

snuckerzlol committed:
Merge pull request #155 from Isabella-L/DG-update

DG update

Isabella-L committed:
refactoring

create a new folder for commands inside command

Isabella-L committed:
checkstyle update

Isabella-L committed:
update command class diagram

LimJiaJing committed:
Correct format of UserGuide and DeveloperGuide. Correct bug in object diagram.

DuckWillow committed:
Merge pull request #157 from LimJiaJing/final-revision

Correct format of UserGuide and DeveloperGuide. Correct bug in object…

snuckerzlol committed:
Updates

snuckerzlol committed:
Merge remote-tracking branch 'origin/master'

snuckerzlol committed:
Merge pull request #156 from Isabella-L/DG-update

Architecture update

snuckerzlol committed:
Merge branch 'AY2122S1-CS2113-T13-4:master' into master

snuckerzlol committed:
Architecture

snuckerzlol committed:
Architecture

snuckerzlol committed:
Architecture

snuckerzlol committed:
Architecture

snuckerzlol committed:
Architecture

snuckerzlol committed:
Architecture

snuckerzlol committed:
Architecture

snuckerzlol committed:
Delete architecture.png

snuckerzlol committed:
Architecture

snuckerzlol committed:
Delete architecture.png

snuckerzlol committed:
Add files via upload

snuckerzlol committed:
Delete architecture.png

snuckerzlol committed:
Add files via upload

snuckerzlol committed:
Update DeveloperGuide.md

DuckWillow committed:
Merge pull request #158 from maxwireddevs/master

Updates

DuckWillow committed:
Update DG and UML diagram

DuckWillow committed:
Update WordLimitGame.java

Isabella-L committed:
Merge pull request #159 from DuckWillow/branch76

Update DG and UML diagram

Isabella-L committed:
update command DG

Isabella-L committed:
udpate

Isabella-L committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-4/tp into DG-update

Isabella-L committed:
update timelimit sequence diagram

Isabella-L committed:
Update UserGuide.md

Isabella-L committed:
update PPP

Isabella-L committed:
update testing

Isabella-L committed:
update testing

snuckerzlol committed:
Merge pull request #162 from Isabella-L/PPP

PPP update by Sicheng

Isabella-L committed:
update UG DG

Isabella-L committed:
Merge branch 'master' into DG-update

Isabella-L committed:
Update UserGuide.md

snuckerzlol committed:
Delete arch.png

snuckerzlol committed:
Add files via upload

snuckerzlol committed:
Delete arch.png

snuckerzlol committed:
Add files via upload

snuckerzlol committed:
Update DeveloperGuide.md

snuckerzlol committed:
Merge pull request #161 from Isabella-L/DG-update

DG and UG update

snuckerzlol committed:
Update AboutUs.md

Isabella-L committed:
Update DeveloperGuide.md

Isabella-L committed:
Merge branch 'DG-update' of https://github.com/Isabella-L/tp into DG-update

Isabella-L committed:
add man feature

Isabella-L committed:
Update UserGuide.md

added man command

Isabella-L committed:
update DG

Isabella-L committed:
Update lusicheng.md

Isabella-L committed:
checkstyle update

Isabella-L committed:
Update DeveloperGuide.md

Isabella-L committed:
udpate readme

LimJiaJing committed:
Merge pull request #163 from Isabella-L/DG-update

DG, UG update, man command feature

Isabella-L committed:
Update UserGuide.md

Isabella-L committed:
Update UserGuide.md

DuckWillow committed:
PPP update

Isabella-L committed:
Set theme jekyll-theme-cayman

DuckWillow committed:
Merge pull request #164 from DuckWillow/branch76

PPP update
